web_programming/daily_horoscope.py

```diff
@@ -0,0 +1,28 @@
+from bs4 import BeautifulSoup as bs
```

Comment:
```suggestion
from bs4 import BeautifulSoup
```
Nobody likes `bs` ;-) so use `BeautifulSoup` instead.